aries_vcx/Cargo.toml

```diff
@@ -31,11 +33,16 @@ serde = "1.0.97"
 serde_json = "1.0.40"
 serde_derive = "1.0.97"
 regex = "1.1.0"
+reqwest = "0.11.10"
```

Comment:
Can't see usage of this

(btw unrelated to your PR - just FYI, we are somewhat misusing `agency-client/src/httpclient.rs`, would like to get rid of that)

Reply:
thnx, deleted this